css/variables.css

```diff
@@ -31,6 +31,7 @@
     --yellow: yellow;
     --red: red;
     --blue: blue;
+    --orange:orange;
```

Comment:
Instead of this, hex code of colors could be given as I suppose that's a better way.

Reply:
Done